packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec

```diff
@@ -0,0 +1,113 @@
+# FIXME:
+#   1. Edit foreman requirement(s) and specify minimum version
+#   2. Delete these lines
+#
+# Generated from foreman_azure_rm-2.0.0.pre1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_azure_rm
+%global plugin_name azure_rm
+%global foreman_min_version FIXME
```

Comment:
Could you fill this in and then remove the top 4 lines?

Reply:
done!